GOVERNANCE.md

```diff
@@ -0,0 +1,111 @@
+to secure modern applications and services.
+
+The 80/20 rule is a core part of the vision behind Keycloak. Stating that 80% of requirements come from roughtly 20%
+of the use-cases. We strongly believe if Keycloak would support all use-cases out of the box it would become bloated 
+and hard to use.
+
+As such Keycloak aims to be opinionated and make it as easy as possible to achieve the common use-cases, while still
+enabling the less common use-cases through custom extensions.
+
+
+## Projects
+
+Keycloak consists of several projects:
+
+* [Keycloak](https://github.com/keycloak/keycloak) - Keycloak Server and Java adapters
+* [Keycloak Documentation](https://github.com/keycloak/keycloak-documentation) - Documentation for Keycloak
+* [Keycloak QuickStarts](https://github.com/keycloak/keycloak-quickstarts) - QuickStarts for getting started with Keycloak
+* [Keycloak Containers](https://github.com/keycloak/keycloak-containers) - Container images for Keycloak
+* [Keycloak Gatekeeper](https://github.com/keycloak/keycloak-gatekeeper) - Proxy service to secure apps and services with Keycloak
+* [Keycloak Node.js Connect](https://github.com/keycloak/keycloak-nodejs-connect) - Node.js adapter for Keycloak
+* [Keycloak Node.js Admin Client](https://github.com/keycloak/keycloak-nodejs-admin-client) - Node.js library for Keycloak Admin REST API
+
+The same governance model applies to all projects. However, the list of maintainers may vary per project. 
+
+
+## Maintainers
+
+The list of maintainers can be found in the [MAINTAINERS.md](MAINTAINERS.md) file in the repository for the individual 
+projects listed in the [Projects](#projects) section.
+
+### Maintainer Responsibilities
+
+A maintainer is someone that has shown deep knowledge of vision, features and codebase. It is their 
+responsibility to drive the project forward, encourage collaboration and contributions, and generally help the 
+community.
+
+Responsibilities of a maintainer includes, but are not limited to:
+
+* Engage in design discussions
+* Actively monitor mailing lists, user forum and chat
+* Contribute high quality code
+* Maintain deep knowledge of vision, features and codebase
+* Review pull requests either themselves or delegate to experts in the relevant area
+* Helping the community
+
+### Becoming a Maintainer
+
+To become a maintainer you need to demonstrate the following:
+
+* Good understanding of vision, features and codebase
+* Contribution of larger features
+* Contribution of bug fixes
+* Participation in design discussions
+* Participation in pull request reviews
+* Ability to collaborate with the team
+* Helping the community
+
+A new maintainer must be proposed by sending an email to the [developer mailing list](https://lists.jboss.org/mailman/listinfo/keycloak-dev).
+The email should include evidence of the above list.
+
+The existing maintainers will then discuss the proposal. If anyone objects, or wants more information, the maintainers 
```

Comment:
No need for a comma after "if anyone objects"

Reply:
Fixing